src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
```

Comment:
Definition of the role of partners can be separated into a separate variable to simplify the readability of the code

Example:
```
 const partnerRole = sex === MALE ? 'wife' : 'husband';
```